db/migrate/20170502034348_create_users.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+# Create users table and apply indices.
+class CreateUsers < ActiveRecord::Migration[5.0]
+  def change
+    create_table :users, id: false do |t|
+      t.serial :id
+      t.string :login, null: false
+      t.string :email, null: false
```

Comment:
Are CockroachDB strings case sensitive? If so is there an option to make them case insensitive?